client/scss/components/_header.scss

```diff
@@ -0,0 +1,289 @@
+//* @define header; weak
+
+.header {
+  padding-top: 0.8rem;
+}
+
+.header__upper-wrap {
+  border-bottom: 1px solid color('keyline-grey');
+}
+
+.header__upper,
+.header__lower {
+  display: flex;
+}
+
+.header__upper {
+  padding-bottom: 0.8rem;
+  justify-content: flex-end;
+  position: relative;
+
+  @include respond-to('medium') {
+    justify-content: space-between;
+  }
+}
+
+.header__lower {
+  justify-content: space-between;
+  position: relative;
+
+  &.is-active {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    background: color('white');
+
+    @include respond-to('medium') {
+      position: relative;
+      width: inherit;
+      height: inherit;
+      background: color('transparent');
+    }
+  }
+}
+
+.header__burger-trigger {
+  position: absolute;
```

Comment:
Yeah, this is a little bit gross isn't it? I was trying to find an acceptable solution for keeping the related show/hide markup together so's not to rely too heavily on JS for e.g. shifting tab focus. But I'm starting to think the latter might be the preferable alternative. I'll knock it up and push it up if I'm happ(y|ier) with it.